quarkus/config-api/src/main/java/org/keycloak/config/BootstrapAdminOptions.java

```diff
@@ -0,0 +1,35 @@
+package org.keycloak.config;
+
+public class BootstrapAdminOptions {
+
+    public static final Option<String> PASSWORD = new OptionBuilder<>("bootstrap-admin-password", String.class)
+            .category(OptionCategory.BOOTSTRAP_ADMIN)
+            .description("Bootstrap admin password")
+            .hidden()
+            .build();
+
+    public static final Option<String> USERNAME = new OptionBuilder<>("bootstrap-admin-username", String.class)
+            .category(OptionCategory.BOOTSTRAP_ADMIN)
+            .description("Username of the bootstrap admin")
+            .hidden()
+            .build();
+
+    public static final Option<Integer> EXPIRATION = new OptionBuilder<>("bootstrap-admin-expiration", Integer.class)
+            .category(OptionCategory.BOOTSTRAP_ADMIN)
+            .description("Time in minutes for the bootstrap admin user to expire.")
+            .hidden()
+            .build();
```

Comment:
We should probably have a default here.

Reply:
It's not necessary since it's a hidden property. I was thinking it was clearer to leave the defaults on the ApplianceBootstrap, but I can certainly move them to the options class if you'd rather.

Reply:
Nah, defaults in `ApplianceBootstrap` work fine.